src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -512,7 +512,8 @@ private static bool IsFloatingPointExpressionOfUnknownPrecision(BoundExpression
                         var mg = (BoundMethodGroup)rewrittenOperand;
                         Debug.Assert(oldNodeOpt.SymbolOpt is { });
```

Comment:
>oldNodeOpt.SymbolOpt [](http://example.com/codeflow?start=37&length=20)

Consider extracting a local.

Reply:
> Consider extracting a local.

I will follow up on this suggestion separately.